quark-test.c

```diff
@@ -330,7 +330,12 @@ fork_exec_nop(void)
 		err(1, "fork");
 	else if (child == 0) {
 		/* child */
-		return (execvp("true", argv));
+		if (relative) {
+			if (execvp("bin/true", argv) == -1)
+				return execvp("./true", argv);
+		}
+		else
+			return execvp("true", argv);
```

Comment:
While testing relative is good, this is a bit too complex, there's no need to copy true around, we can do the same with chdir, I'll push a change